terraform/modules/waf/main.tf

```
@@ -21,6 +21,7 @@ locals {
     bad_inputs    = replace("${var.prefix}cloudfront-waf-bad-inputs", "-", "")
     ip_reputation = replace("${var.prefix}cloudfront-waf-ip-reputation", "-", "")
     ip_allowlist  = replace("${var.prefix}cloudfront-waf-ip-allowlist", "-", "")
+    ip_blocklist  = replace("${var.prefix}cloudfront-waf-ip-blacklist", "-", "")
```

should be consistent
```suggestion
    ip_blocklist  = replace("${var.prefix}cloudfront-waf-ip-blocklist", "-", "")
```